src/app/badges/page.tsx

```diff
@@ -26,46 +27,34 @@ export default function Badges() {
           {error && <div>Error...</div>}
           {badges && (
             <>
-              <Stack
+              <Box
```

Comment:
why box?

Reply:
no reason to change it to box when Stack is essentially a box with direction and we aren't using a direction here.

Reply:
no strong reference here.

Reply:
stack has some default styles I rather not need to override